Content.Server/GameObjects/Components/Atmos/GasAnalyzerComponent.cs

```diff
@@ -32,7 +27,9 @@ public class GasAnalyzerComponent : SharedGasAnalyzerComponent, IAfterInteract,
         private BoundUserInterface _userInterface = default!;
         private GasAnalyzerDanger _pressureDanger;
         private float _timeSinceSync;
-        private const float TimeBetweenSyncs = 10f;
+        private const float TimeBetweenSyncs = 2f;
+        private bool _checkPlayer = false; // Check at the player pos or at some other tile?
+        private Vector2 _offset; // The direction in which you're holding the analyzer
```

Comment:
Why do you store an offset? instead of, y'know, the other position?

Reply:
My ideas if the player moved out of reached of the original tile (because you don't want him to scan the gas on the other side of the station):
* Close the window
* Use the local player position
* Use the position where he "pointed" the gas analyzer

I chose the last one

Reply:
But you set the offset before you open the UI...

Just store the `_analyzedPosition` (or some such), and set it as an argument to `OpenInterface()`
then in `CloseInterface()` you null out the position. This covers both the player's position and a 'clicked' position nicely.